src/orga/subteams.rst

```diff
@@ -2,6 +2,66 @@ A list of current sub-teams
 ***************************
 
 
+Code of Conduct Sub-Team
+=============================
+
+Role
+----
+The code-of-conduct (CoC) sub-team is responsible for accepting, handling
+and resolving code-of-conduct related items (including but not limited to
+CoC violations, enforcement, communications, warnings, etc.) on conda-forge's
+communication channels (including but not limited to Gitter, GitHub, Keybase, the various
+mailing lists, etc).
+
+Charter
+-------
+Static
+
+Responsibility
+--------------
+The CoC sub-team will operate via the following rules:
+
+- It can only be comprised of ``core`` members.
+- Any ``core`` member is welcome to participate.
+- ``core`` members involved in CoC reports will use their best judgment to recuse
+  themselves in cases where there may be or even appear to be a conflict of interest.
+- The CoC team will judge CoC-related issues according to the current official
+  conda-forge CoC. The ``conda-forge`` CoC is subject to change by the wider ``core`` group.
+- The team may seek guidance from NumFocus and the wider ``core`` group as needed.
+- Due to the potentially sensitive nature of CoC violations, the CoC team will
+  strive to preserve the confidentiality of involved persons as much as possible.
+- The CoC team may issue warnings through e-mail, gitter, github etc. in order to
+  prevent situations from escalating.
+- The CoC team will maintain records of CoC reports, the CoCs findings in each case, and
+  will report them as appropriate to the wider core group promptly.
+- The CoC team will maintain secured reporting channels for potential CoC violations.
+- The CoC team may temporarily remove maintainers for a period of up to one month
+  from feedstocks through the "yes" vote of at least three CoC team members and no "no" votes.
```

Comment:
Is this dependent on the number of CoC team members? (eg if the group member ship drops below three, is this no longer actionable?)